core/parachain/backing/grid.hpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <numeric>
+#include <unordered_set>
+#include <vector>
+
+#include "crypto/hasher/hasher_impl.hpp"
+#include "parachain/backing/chacha.hpp"
+
+namespace kagome::parachain::grid {
+  struct Grid {
```

Comment:
Please drop few comments about structure, or add link to reference implementation (if any)